dev-cpp/sourcetrail/sourcetrail-2018.3.13.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop gnome2-utils xdg-utils
+
+DESCRIPTION="A cross-platform source explorer for C/C++ and Java"
+HOMEPAGE="https://www.sourcetrail.com/"
+SRC_URI="https://www.sourcetrail.com/downloads/${PV}/linux/64bit -> ${P}.tar.gz"
+
+LICENSE="Sourcetrail"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="dev-util/patchelf"
+
+S="${WORKDIR}/Sourcetrail"
+QA_PREBUILT="*"
+
+src_install() {
+	# This removes the rpath entries with $$ORIGIN
+	# since they trigger warnings when merging
+	for f in "Sourcetrail" "sourcetrail_indexer"; do
+		rpath=$(patchelf --print-rpath "${f}" 2>/dev/null)
+		new_rpath=${rpath//\$\$ORIGIN\/lib\/:/}
+		patchelf --set-rpath "${new_rpath}" "${f}" || die
+	done
+
+	insinto /opt/sourcetrail
+	doins -r EULA.txt README data lib plugin user
```

Comment:
BTW - The user directory actually contains some sample projects, would it make sense to make these optional with a USE flag? And if so, do we have a standard USE flag for this?

Reply:
If they don't take much space, probably not worth the effort. The standard flag is USE=examples but an explanatory local def is usually worthwhile

Reply:
It actually takes up 12MB, so I think it is worth the effort. Let me just update the pull request :)

Reply:
I have added this to the updated PR.